here-naksha-lib-jbon/src/commonMain/kotlin/IMapApi.kt

```diff
@@ -98,4 +98,11 @@ interface IMapApi {
      * @throws IllegalArgumentException If the given map is no native map.
      */
     fun iterator(map: IMap): IMapIterator
+
```

Comment:
If it adds, can we can we call it as well plus (which is the defined operator overloading name)?